lib/kanal/plugins/user_system/user_system_plugin.rb

```diff
@@ -10,6 +11,17 @@ module UserSystem
       # registration and removal system
       class UserSystemPlugin < Kanal::Core::Plugins::Plugin
         include Models
+        include Helpers
+
+        attr_reader :auto_create
+
+        #
+        # @param [Boolean] auto_create Specify if auto-creation of users on incoming message will be used
+        #
+        def initialize(auto_create: false)
```

Comment:
This serves wrong purpose because "auto_create: true" in constructor argument - implies there will be something automatically created in plugin with just enabling it inside constructor arguments.

What is purpose of optional subclass creation in this case?

Reply:
Maybe rename it to with_auto_creator? Since the class is calledAutoCreator

Reply:
> Maybe rename it to with_auto_creator? Since the class is called AutoCreator

What is the reasoning behind making auto_creator optional?

Reply:
Hmm. Maybe someone doesn't want to create users automatically? But then, the point of the issue was to make autocrearion buit-in, so maybe it should be just enabled. Will do.